core/Pos/Util/Modifier.hs

```diff
@@ -53,6 +61,11 @@ instance (Eq k, Hashable k) =>
 
 instance (Eq k, Hashable k) => Semigroup (MapModifier k v)
 
+instance (Buildable k, Buildable v) => Buildable (MapModifier k v) where
+    build mm =
+      bprint ("MapModifier { deletions "%listJson%", insertions: "%listJson%"}")
+      (deletions mm) (map (bprint pairF) (insertions mm))
```

Comment:
I don't know who was adding this instance, but FYI: there is `mapBuilder` and `mapJson` formatter.